### PR TITLE
Add discussion to acceleo repository

### PR DESCRIPTION
**Pull request by heurtematte**
Related to https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/4771

### CODE REVIEW

**Review by netomi — APPROVED** [review]

**Comment by ylussaud** [comment]
I can't merge this pull request because I don't have write access to the `.eclipsefdn` repository...

**Comment by netomi** [comment]
you are not supposed to merge directly in this repo. There is an automerge feature which allows you to merge most PRs by adding a comment `/otterdog merge` . You will be instructed if this is possible.

**Pull request merged**
